src/data/faq.json

```
@@ -316,7 +316,7 @@
                         "anchor": "vac_cert_relatives",
                         "active": true,
                         "textblock": [
-                            "With the Corona-Warn-App, you can only scan and add vaccination certificates from one person. It is not possible to add your own and additionally, for example, that of your child or partner.<br>With the CovPass-App, the digital vaccination certificates of children or partners can be stored together on a smartphone."
+                            "Starting with version 2.5 of the Corona-Warn-App, you can scan and add proof of vaccination for several people, such as that of your child or partner.<br>Also with the CovPass-App, the digital vaccination certificates of children or partners can be stored together on a smartphone."

```

```suggestion
                            "Starting with version 2.5 of the Corona-Warn-App, you can scan and add proof of vaccination for several people, such as that of your child or partner.<br>Also with the CovPass-App, the digital vaccination certificates of several people can be stored together on a smartphone."
```
Otherwise it looks like CovPass-App can only store the certificates of children or partners. It could be anyone though.